inc/dbmysql.class.php

```diff
@@ -57,42 +52,33 @@ class DBmysql {
    //! Database Error
    public $error              = 0;
 
-   /// Slave management
+   // Slave management
    public $slave              = false;
    /** Is it a first connection ?
     * Indicates if the first connection attempt is successful or not
     * if first attempt fail -> display a warning which indicates that glpi is in readonly
    **/
    public $first_connection   = true;
-   /// Is connected to the DB ?
+   // Is connected to the DB ?
    public $connected          = false;
 
 
    /**
     * Constructor / Connect to the MySQL Database
     *
-    * try to connect
-    *
-    * @param $choice integer, host number (default NULL)
-    *
-    * @return nothing
```

Comment:
The [phpDocumentor](https://phpdoc.org/docs/latest/references/phpdoc/tags/return.html#description) explain :
```
It is RECOMMENDED when documenting to use this tag with every function and method. Exceptions to this recommendation are:
  1. constructors, the @return tag MAY be omitted here, in which case @return self is implied.
  2. functions and methods without a `return` value, the @return tag MAY be omitted here, in which case @return void is implied.
```

I think we are on the 1. case ?